src/Symfony/Component/Console/Descriptor/ApplicationDescription.php

```diff
@@ -134,15 +134,17 @@ private function inspectApplication()
     private function sortCommands(array $commands)
     {
         $namespacedCommands = array();
+        $globalCommands = array();
         foreach ($commands as $name => $command) {
             $key = $this->application->extractNamespace($name, 1);
             if (!$key) {
-                $key = '_global';
+                $globalCommands['_global'][$name] = $command;
```

Comment:
I suggest the ``_global`` key here, build ``$globalCommands`` as being only the array of global commands themselves, and the prepending this array in the list of commands after the sorting with ``array_shift($namespacedCommands, $globalCommands)`` (and doing it only in case the list of global commands is not empty to avoid useless work)